CSharp_Course_Project/Task2_StudentDictionary/StudentDictionary.cs

```diff
@@ -0,0 +1,106 @@
+﻿using CSharp_Course_Project.Task1_StudentList;
+using System;
+using System.Collections.Generic;
+
+namespace CSharp_Course_Project.Task2_StudentDictionary
+{
+    public class StudentDictionary
+    {
+        private readonly Dictionary<string, Student> students = new Dictionary<string, Student>();
+
+        public void ManageStudentDictionary()
+        {
+            while (true)
+            {
+                Console.WriteLine($"Please, choose one of the option:{Environment.NewLine}" +
+                $"1 - Add student{Environment.NewLine}" +
+                $"2 - Remove student{Environment.NewLine}" +
+                $"3 - Print student list{Environment.NewLine}" +
+                $"0 - Escape{Environment.NewLine}");
+                var option = Console.ReadLine();
+
+                switch (option)
+                {
+                    case "1":
+                        this.AddStudent();
+                        break;
+                    case "2":
+                        this.RemoveStudent();
+                        break;
+                    case "3":
+                        this.PrintStudent();
+                        break;
+                    case "0":
+                        break;
+                    default:
+                        Console.WriteLine("Wrong option. Please, try again...");
+                        continue;
+                }
+
+                if (option == "0")
+                    break;
+            }
+        }
+
+        public void AddStudent()
+        {
+            while (true)
+            {
+                Console.Write("Please enter a student name: ");
+                var name = Console.ReadLine();
+
+                Console.Write("Please enter a student surname: ");
+                var surname = Console.ReadLine();
+
+                Console.Write("Please enter a student course (1-5): ");
+                var course = Byte.TryParse(Console.ReadLine(), out byte courseNumber);
+
+                Console.Write("Please enter a student group: ");
+                var group = Console.ReadLine();
+
+                Console.Write("Please enter a student faculty: ");
+                var faculty = Console.ReadLine();
+
+                var student = new Student(name, surname, courseNumber, group, faculty);
+                this.students.Add(student.Name + " " + student.Surname, student);
```

Comment:
Why don't you use string interpolation, it is simpler for reading i think.

Reply:
I used it in StudentStack and StudentQueue. Fixed it.